src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Doctrine;
+
+use Doctrine\DBAL\Connection as DriverConnection;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\Synchronizer\SingleDatabaseSynchronizer;
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * @author Vincent Touzet <vincent.touzet@gmail.com>
+ */
+class Connection
+{
+    /** @var string */
+    private $tableName;
+    /** @var \Doctrine\DBAL\Driver\Connection */
```

Comment:
It can be removed, same above and below.

Reply:
Ok no docs for properties ? I can remove in other file altough

Reply:
Yeah, at least for those initialized in the constructor.